web/app/js/components/StatusTable.jsx

```diff
@@ -70,7 +71,12 @@ const columns = {
         });
       }
     };
-  }
+  },
+  grafana: ConduitLink => {
```

Comment:
Sorry for only thinking of this just now, but I was playing around with different placements of the grafana icon: wdyt about placing the icon at the end of each name in the first column, like you do for the deployment headers? that way all the links in the table are close by, and we avoid the grafana icons looking stranded when the column is wide?

```
      render: row => {
        let name = row.name;
        return (<React.Fragment>
          {!resource.url ? name : <ConduitLink to={`${resource.url}${name}`}>{name}</ConduitLink>}
          {row.added ? <span>&nbsp;<GrafanaLink name={row.name} size={16} conduitLink={ConduitLink} /></span> : null}
        </React.Fragment>);
      }
```

![screen shot 2018-04-04 at 2 35 45 pm](https://user-images.githubusercontent.com/549258/38336517-dcf2b826-3816-11e8-984c-7b9f53236abf.png)

<img width="1002" alt="screen shot 2018-04-04 at 2 52 18 pm" src="https://user-images.githubusercontent.com/549258/38336765-d0a41668-3817-11e8-9f14-e8876029019c.png">

Reply:
Yeah I like how that looks way better, I'll make the change.

I had initially thought it might be confusing that the text and the icon link to different things, but seeing your screenshot I think it looks fine.